app/src/main/res/values/strings.xml

```diff
@@ -286,79 +288,79 @@
         <item>Full</item>
     </string-array>
     <string name="pref_name_autoSyncQueue_enabled">Separate auto-sync for local changes</string>
-    <string name="pref_desc_autoSyncQueue_enabled">This allows local changes (added links, archived articles, etc.) to be synced to the server as soon as network becomes available.</string>
+    <string name="pref_desc_autoSyncQueue_enabled">Syncs local changes (added links, archived articles, etc.) to the server once online again.</string>
     <string name="pref_name_imageCache_enabled">Put article images in cache</string>
-    <string name="pref_desc_imageCache_enabled">Automatically downloads all images from every article and saves them to the device. Depending on the length of your wallabag article list, this can take up a lot of time and space.</string>
+    <string name="pref_desc_imageCache_enabled">Auto-download all images from every article onto the device. (Takes up a lot of time and space if the article list is long.)</string>
 
     <string name="pref_name_storage_dbPath">Database location</string>
     <string name="pref_name_storage_dbPath_internalStorage">Internal storage</string>
     <string name="pref_name_storage_dbPath_externalStorage">External storage</string>
     <string name="pref_name_storage_dbPath_dbMoved">Database moved</string>
 
-    <string name="pref_categoryName_miscellaneous">Miscellaneous</string>
+    <string name="pref_categoryName_miscellaneous">Misc</string>
     <string name="pref_name_misc_appendWallabagMention">Append wallabag mention</string>
     <string name="pref_desc_misc_appendWallabagMention">Append \"via @wallabagapp\" to shared texts</string>
     <string name="pref_name_misc_handleHttpScheme">Handle HTTP scheme</string>
-    <string name="pref_desc_misc_handleHttpScheme">Show wallabag in the list of web browsers</string>
+    <string name="pref_desc_misc_handleHttpScheme">Show wallabag in the list of web-browsers</string>
     <string name="pref_name_misc_wipeDB">Wipe Database</string>
-    <string name="pref_desc_misc_wipeDB">Wipes all the articles from local database, also removes all not synchronized local changes and URLs</string>
+    <string name="pref_desc_misc_wipeDB">Deletes articles and all unsynced local changes and URLs from the local database.</string>
     <string name="pref_name_misc_wipeDB_confirmTitle">Wipe Database?</string>
-    <string name="pref_name_misc_wipeDB_confirmMessage">Are you sure you want to wipe the database?</string>
+    <string name="pref_name_misc_wipeDB_confirmMessage">Are you sure you want to remove the database?</string>
```

Comment:
"Clear" would probably be a better word.

Reply:
Or "empty", where one is more normal, and the other more correct".

Reply:
clear or empty is fine here as it is not removed.